### ISSUE TITLE
Are Lohit Fonts available in bold and italic ?

### ISSUE THREAD

**Issue by Shreeshrii** [pravins/lohit2#40]
Ref: http://tug.org/pipermail/xetex/2015-July/026073.html

[XeTeX] Tamil Unicode fonts (bold, italic) for XeLaTeX

P. P. Narayanaswami swami at mun.ca 
Sat Jul 18 17:13:42 CEST 2015

I want to type Tamil text in Ubuntu 14.04, with  TeXLive 2014.
The Lohit Tamil font that comes with the Ubuntu is good normal font.
But since I cannot use font shape commands like \textbf, \slshape, etc
with this font in XeLaTeX, I need good bold and italic Tamil fonts to match
Lohit Tamil fonts. Also some fonts do not have the proper glyphs for "shrI",
"kSha" and  "sha" in Tamil.

Can anyone, who is using XeLaTeX in Tamil, suggest three good matching Tamil
fonts, normal, bold, and italic shapes that also contain the glyphs "shrI",
"kSha, and "sha" to be used in the same document along with English (also bold,
italic and normal)?  Thanks in advance.

**Comment by pravins** [comment]
Yes, Unfortunately Lohit Tamil is available only in Regular variant. Good to drop email on lohit-devel list. You can get better answers there.

**Comment by santhoshtr** [comment]
As an alternate font, you may check https://github.com/santhoshtr/meera-tamil but it also does not have bold and italic versions

**Comment by davelab6** [comment]
Karla by Jonny Pinhorn has all 4 styles
